users/tests/test_views.py

```diff
@@ -1,13 +1,100 @@
+import datetime
+
 from django.contrib.auth import get_user_model
 from django.contrib.auth.models import Group, Permission
 from django.test import TestCase
+from django.urls import reverse
+
+from feedbacks.tests.mixins import SetUpUserMixin
+
+__all__ = ['LoginViewTest', 'RegisterViewTest', 'UpdateUserViewTest',
+           'PasswordChangeViewTest', 'TestViews']
+
+
+class LoginViewTest(SetUpUserMixin, TestCase):
+
+    def setUp(self):
+        self.client.login(username='test_user', password='12345')
+
+    def test_redirect_when_user_logged_in(self):
+        response = self.client.post(reverse('login'), {'username': 'test_user',
+                                                       'password': '12345'})
+
+        self.assertEqual(response.status_code, 302)
+
+    def test_redirect_to_home_page_if_user_logged_in(self):
+        response = self.client.get(reverse('login'))
+
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(response, reverse('home'))
+
+
+class RegisterViewTest(SetUpUserMixin, TestCase):
+
+    def setUp(self):
+        self.client.login(username='test_user', password='12345')
+
+    def test_register_view(self):
+        response = self.client.post(reverse('register'), {
+            'username': 'Yurii',
+            'email': 'jura@mail.com',
+            'first_name': 'Yurii',
+            'last_name': 'Kulyk',
+            'password1': 'testview123',
+            'password2': 'testview123',
+        })
+        self.assertEqual(response.status_code, 302)
+        self.assertEqual(get_user_model().objects.get(username='Yurii').email,
+                         'jura@mail.com')
+
+    def test_redirect_to_home_page_if_user_logged_in(self):
+        self.client.login(username='test_user', password='12345')
+        response = self.client.get(reverse('register'))
+
+        self.assertEqual(response.status_code, 302)
+        self.assertRedirects(response, reverse('home'))
+
+
+class UpdateUserViewTest(SetUpUserMixin, TestCase):
+
+    def setUp(self):
+        self.client.login(username='test_user', password='12345')
+
+    def test_valid_data_update_view(self):
+        self.client.login(username='test_user', password='12345')
+        current_user = get_user_model().objects.get(username='test_user')
+        response = self.client.post(reverse('profile-update'),
+                                    {
+                                        'first_name': 'Yurii',
+                                        'last_name': 'Kulyk',
+                                        'experience': 1,
+                                        'date_of_birth': datetime.date(
+                                            1999, 5, 21),
+                                        'education': 'Secondary',
+                                        'gender': 'Male', })
+
```

Comment:
Can you create variable UPDATE_PROFILE_DATA into your tests_data.py and use it here?